app/components/expense-filter.tsx

```diff
@@ -0,0 +1,24 @@
+import { ButtonPrimary } from "./button-primary";
+import { SelectPrimary } from "./select-primary";
+
+const ExpenseFilter = () => {
+  return (
+    <div className="py-4 flex justify-between items-center">
+      <div>
+        <ButtonPrimary name="Received" />
+        <ButtonPrimary name="Submmitted" />
+      </div>
+      <div className="flex space-x-4">
+        <input type="text" placeholder="Search" className="input" />
```

Comment:
```suggestion
        <input type="text" placeholder="Search" className="input border rounded-lg p-2" />
```
Enhance search input a bit by adding a border and padding

Reply:
updated